documentation/testing_plans.md

```diff
@@ -272,6 +272,7 @@ documentation.
 | [`apply_prep`](plan_functions.md#apply-prep) | [`allow_apply_prep`](boltspec_reference.md#allow-apply-prep) | - |
 | [`download_file`](plan_functions.md#download-file) | [`allow_download`](boltspec_reference.md#allow-download) | [`expect_download`](boltspec_reference.md#expect_download) |
 | [`out::message`](plan_functions.md#out::message) | [`allow_out_message`](boltspec_reference.md#allow-out-message) | [`expect_out_message`](boltspec_reference.md#expect-out-message) |
+| [`out::verbose`](plan_functions.md#out::verbose) | [`allow_out_verbose`](boltspec_reference.md#allow-out-verbose) | [`expect_out_verbose`](boltspec_reference.md#expect-out-verbose) |
```

Comment:
```suggestion
| [`out::verbose`](plan_functions.md#outverbose) | [`allow_out_verbose`](boltspec_reference.md#allow-out-verbose) | [`expect_out_verbose`](boltspec_reference.md#expect-out-verbose) |
```

Dita will remove the colons here. While we're here, we should also update the link for `out::message` above to not have colons.

Reply:
Also, `allow_out_verbose` and `expect_out_verbose` both need to be added to the Boltspec reference page: https://github.com/puppetlabs/bolt/blob/main/documentation/boltspec_reference.md

Reply:
On it